types/react/test/elementAttributes.tsx

```diff
@@ -2,62 +2,62 @@ import * as React from 'react';
 
 const testCases = [
     <span />,
-    <span accessKey="s" />,
+    <span accessKey='s' />,
     <span autoFocus />,
-    <span className="klass" />,
+    <span className='klass' />,
     <span contentEditable />,
-    <span contextMenu="menuId" />,
-    <span dir="rtl" />,
+    <span contextMenu='menuId' />,
+    <span dir='rtl' />,
     <span draggable />,
     <span hidden />,
-    <span id="s" />,
-    <span lang="art-x-tokipona" />,
-    <input placeholder="placeholder" />,
-    <span slot="my-text" />,
+    <span id='s' />,
+    <span lang='art-x-tokipona' />,
+    <input placeholder='placeholder' />,
+    <span slot='my-text' />,
     <span spellCheck />,
     <span tabIndex={0} />,
-    <span title="title" />,
-    <span role="button" />,
-    <span autoCapitalize="off" />,
-    <span autoCapitalize="none" />,
-    <span autoCapitalize="on" />,
-    <span autoCapitalize="words" />,
-    <span autoCapitalize="sentences" />,
-    <span autoCorrect="off" />,
-    <span autoCorrect="on" />,
-    <span translate="no" />,
-    <span translate="yes" />,
+    <span title='title' />,
```

Comment:
Hm, there's something wrong here. We should be setting `jsx.quoteStyle` to double in the config if we are still setting `quoteStyle` to single.